test/new-e2e/tests/agent-platform/common/process/process.go

```diff
@@ -0,0 +1,37 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-present Datadog, Inc.
+
+// Package process provides utilities for testing processes
+package process
+
+import (
+	"fmt"
+
+	"github.com/DataDog/datadog-agent/test/new-e2e/pkg/components"
+	"github.com/DataDog/datadog-agent/test/new-e2e/tests/windows"
+	componentos "github.com/DataDog/test-infra-definitions/components/os"
+)
+
+// IsProcessRunning returns true if process is running
+func IsProcessRunning(host *components.RemoteHost, process string) (bool, error) {
```

Comment:
<!-- Suggestions propose improvements to the current subject. It’s important to be explicit and clear on what is being suggested and why it is an improvement. -->
**💬 suggestion**
As earlier, we know it is a `process` by the package name
```suggestion
func IsRunning(host *components.RemoteHost, processName string) (bool, error) {
```